worker/base/worker.go

```diff
@@ -90,6 +90,10 @@ type Worker struct {
 
 // NewWorker instantiates a local worker
 func NewWorker(opt WorkerOpt) (*Worker, error) {
+	return NewWorkerWithContext(context.TODO(), opt)
+}
+
+func NewWorkerWithContext(ctx context.Context, opt WorkerOpt) (*Worker, error) {
```

Comment:
Just change the signature

Reply:
may break other pkgs which use `buildkit`.

Reply:
Then they update. We don't care about go API stability for these packages where it adds maintenance cost.